src/components/autocomplete/autocomplete-theme.scss

```diff
@@ -13,6 +13,44 @@ md-autocomplete.md-THEME_NAME-theme {
       background: '{{background-600-0.3}}';
     }
   }
+  &.md-accent {
+    md-input-container.md-input-focused {
+      .md-input {
+        border-color: '{{accent-color}}';
+      }
+      label,
+      md-icon {
+        color: '{{accent-color}}';
+      }
+    }
+    md-progress-linear {
+      .md-container {
+        background-color: '{{accent-100}}';
+      }
+      .md-bar {
+        background-color: '{{accent-color}}';
+      }
+    }
+  }
+  &.md-warn {
+    md-input-container.md-input-focused {
+      .md-input {
+        border-color: '{{warn-A700}}';
+      }
+      label,
+      md-icon {
+        color: '{{warn-A700}}';
+      }
+    }
+    md-progress-linear {
+      .md-container {
+        background-color: '{{warn-100}}';
+      }
+      .md-bar {
+        background-color: '{{warn-color}}';
+      }
+    }
+  }
```

Comment:
@DevVersion not sure if you mean in autocomplete-theme.scss or if you mean on in input and linear progress that could be included?

Was hoping that the mixins used in md-tabs would have been a good starting point -- upon closer inspection they just repeated the same code in mixins: https://github.com/angular/material/blob/317c1c8d95bca192397336fc0f65dab402c23680/src/components/tabs/tabs-theme.scss#L1  <- were you looking for something more like that?